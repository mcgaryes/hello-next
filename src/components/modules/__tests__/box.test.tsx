import {render} from '@testing-library/react'
import Box from "@/modules/box";

test('Renders box unchanged', () => {
    const {container} = render(<Box />)
    expect(container).toMatchSnapshot()
})
