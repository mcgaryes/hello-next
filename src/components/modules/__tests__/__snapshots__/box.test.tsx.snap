// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders box unchanged 1`] = `
<div>
  <div />
</div>
`;
